### PR TITLE
Refactor=sheet fetch

### PR DESCRIPTION
**Pull request by sserbula**

### CODE REVIEW

**Review by nikolicdusan — CHANGES_REQUESTED** [review]
Please review comments

File: Exebite.Common/Extensions/StringExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Exebite.Common.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string FirstCharToUpper(this string input) =>
+            input switch
+            {
+                null => throw new ArgumentNullException(nameof(input)),
+                "" => throw new ArgumentException($"{nameof(input)} cannot be empty", nameof(input)),
```
[the remark this answers is not in the document]
Comment (sserbula) [reply Exebite.Common/Extensions/StringExtensions.cs:12]: `string.IsNullOrEmpty()` returns boolean value and I am using `string` inside `switch`.
When I tried using `string.Empty` I got error `A Constant value is expected`

File: Exebite.Common/Extensions/StringExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                _ => input.First().ToString().ToUpper() + input.Substring(1)
```
[the remark this answers is not in the document]
Comment (sserbula) [reply Exebite.Common/Extensions/StringExtensions.cs:13]: If number or symbol is the first char, nothing happens, same string is returned, e.g. `22s` => `22s`

File: Exebite.Common/Constants/SheetConstants.cs [hunk 3]
```diff
@@ -25,5 +25,41 @@ public class SheetConstants
...
+        public const string HEYDAY_FOOD_DESERT = "desert";
+        public const string HEYDAY_FOOD_SMOOTHIES = "smoothies";
+        public const string HEYDAY_FOOD_WELLNESS_SHOTS = "wellness shots";
+        public const short HEYDAY_MEAL_TYPE_ROW = 1;
+
+        public const string LIPA_MENU_NAME_RANGE = "!D1:1";
+        public const string LIPA_MENU_PRICE_RANGE = "!D4:4";
+
+        public const string TOPLI_BROK_MENU_NAME_RANGE = "!D1:1";
+        public const string TOPLI_BROK_MENU_PRICE_RANGE = "!D4:4";
```
Comment (bjokic) [inline Exebite.Common/Constants/SheetConstants.cs:45]: Correct the constant name

**Review by bjokic — CHANGES_REQUESTED** [review]
Please review the comment.

**Review by bjokic — APPROVED** [review]
Looks good

**Review by nikolicdusan — APPROVED** [review]

**Pull request merged**